### PR TITLE
New Rule - Do you know how to allocate the expenses to the offices?

### PR DESCRIPTION
**Pull request by JimmyChenSSW**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

A task from @stephanfakossw that create a rule to explain how we allocate expenses to different offices.

> 2. What was changed?

- New rule created
- Update the category of better accounting

> 3. Did you do pair or mob programming (list names)?

@stephanfakossw

### CODE REVIEW

File: rules/allocate-the-expenses-to-the-offices/rule.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+guid: 596829da-9013-402d-b8f7-c34a6d42f411
+---
+
+When a company has multiple offices in different locations, it is important to allocate expenses to each office accurately to ensure that the financial reports reflect the true costs of running the business in each location.
+Proper expense allocation can also help identify areas of inefficiency and improve cost management.
+
+However, allocating expenses to offices in different locations can be challenging, especially when the expenses are shared among different offices or are difficult to track. 
+It requires a systematic approach and a set of guidelines to ensure that expenses are allocated fairly and accurately.
+
+![](Allocate-expense-1.jpg)
```
Comment (sethdaily) [inline rules/allocate-the-expenses-to-the-offices/rule.md:22]: Add a caption

File: rules/allocate-the-expenses-to-the-offices/rule.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+If an expense is attributable to a specific office, it should be allocated directly to that office. 
+This includes expenses such as travel expenses incurred by an employee visiting a specific office or recruitment costs for a particular job opening in a specific office.
+
+E.g., if there is a recruitment ad for an open position in Melbourne with a cost of $1,000, it should be allocated to the Melbourne office.
+
+The office chosen should genuinely reflect the beneficiary.
+
+* If Bob travels from Sydney to Brisbane to give a speech at a conference held by the Brisbane office, the cost should be allocated to Brisbane.
+* If Bob travels from Sydney to Melbourne for training, then the cost is allocated to Sydney.
```
Comment (sethdaily) [inline rules/allocate-the-expenses-to-the-offices/rule.md:34]:
```suggestion
::: greybox
* If Bob travels from Sydney to Brisbane to give a speech at a conference held by the Brisbane office, the cost should be allocated to Brisbane.
* If Bob travels from Sydney to Melbourne for training, then the cost is allocated to Sydney.
:::

::: good
Figure: Good example - always allocate expenses to the more relevant office
:::
```

File: rules/allocate-the-expenses-to-the-offices/rule.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+E.g., if there is a recruitment ad for a position advertised for any office, the cost is not directly allocated to a specific office but is instead allocated by an end-of-month adjustment.
+
+Unassigned expenses will be tracked separately and allocated to each office by an end-of-month adjustment. See the figure below. The allocation frequency should be clearly defined in the company rule.
+Expenses from the company level can be allocated to each office based on the admin rate which can be calculated by different methods, e.g., cost of sales.
+
+* These expenses will initially be treated as unassigned when created.
+* A manual journal will be created by the end of the reporting period.
+
+![✅ Good Example: Software expenses from the company level would be assigned to different offices based on the admin rate](keep-yourself-connected-3.png)
```
Comment (sethdaily) [inline rules/allocate-the-expenses-to-the-offices/rule.md:54]:
```suggestion
::: good
![Figure: Good example - Software expenses from the company level would be assigned to different offices based on the admin rate](keep-yourself-connected-3.png)
:::
```

File: rules/allocate-the-expenses-to-the-offices/rule.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+### Review and Communication 
+
+The allocation process should undergo a review by someone with accounting expertise to ensure the accuracy and consistency of the allocation results. 
+Any errors or inconsistencies discovered during the review should be identified and promptly corrected.
```
Comment (sethdaily) [inline rules/allocate-the-expenses-to-the-offices/rule.md:59]: I don't think this is necessary - obvious
```suggestion
```

File: rules/allocate-the-expenses-to-the-offices/rule.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+The company rules governing the allocation process should be effectively communicated to all relevant parties. 
+This includes accounting staff, managers of each office, and any other employees involved in the expense tracking and reporting process. 
+This communication is essential to ensure that everyone is on the same page and understands their role in the process.
```
Comment (sethdaily) [inline rules/allocate-the-expenses-to-the-offices/rule.md:63]: I don't think you need these - obvious
```suggestion
```

**Review by sethdaily — CHANGES_REQUESTED** [review]
1. See comments
2. I would change the title from:
Do you know how to allocate the expenses to the offices?
to
Do you know how to allocate expenses per office?

File: rules/allocate-the-expenses-to-the-offices/rule.md (hunk 1 above)
Comment (JimmyChenSSW) [reply rules/allocate-the-expenses-to-the-offices/rule.md]: Updated using new figure

**Review by JimmyChenSSW — COMMENTED** [review]
Changed as per suggested by Seth

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**